### ISSUE TITLE
`\c` not working as expected

### ISSUE THREAD

**Issue by MarcelRobitaille** [amaanq/tree-sitter-doxygen#8]
First of all, thank you for the amazing extension :heart:

I noticed that `\c` is not being parsed correctly I think. I would expect the next word to be interpreted as code. However, if I look into the treesitter playground, I see:

```
            tag [61, 54] - [61, 91]           <-- `\c` up to the end of the line
              tag_name [61, 54] - [61, 56]    <-- `\c\`
              description [61, 57] - [61, 91] <-- everything after `\c` until the end of the line

```

I don't really care if it looks like text, but I really want to ignore these code words from spelling. Obviously, function names are not real words. It makes `set spell` very annoying.

The same thing is happening for the word after `\param[in]`, but in this case, it's getting its own identifier. I'm guessing this isn't a parsing issue but rather an injections issue on the nvim side?

I remember ignoring certain highlight groups from spelling from within vim, but I think it would first need to be parsed, right?

What do you think?

**Comment by amaanq** [comment]
Hm, I see, thanks for pointing it out. Should be fixed once nvim-ts updates the lockfile

**Comment by MarcelRobitaille** [comment]
I just updated nvim-ts. I did `:PackerUpdate` and then `TSUpdate`. The result is that now, `\c` isn't recognized at all. It used to be blue and had the tree I shared above. Now, I get this:

![image](https://github.com/amaanq/tree-sitter-doxygen/assets/8503756/d0cc79ae-6249-4f12-af50-f372ce594fbc)

`\c` is not recognized and the argument after `\param[in]` is still showing a spelling mistake.
